alm/package.json

```diff
@@ -30,11 +31,12 @@
     "web3-eth-contract": "1.2.7"
   },
   "scripts": {
-    "start": "./load-env.sh react-app-rewired start",
-    "build": "./load-env.sh react-app-rewired build",
+    "start": "yarn createSnapshots && ./load-env.sh react-app-rewired start",
```

Comment:
will it work with the new dockerfile which uses `serve`?

Reply:
Yes, the script to create the snapshot is also called in the `build` script. It will generate the snapshots and include them in the sources of the app in the folder `src/snapshots` so they will be included in the code of the built app
https://github.com/poanetwork/tokenbridge/blob/2e062ff1d715229520a878a538cabdf9842e7124/alm/package.json#L35

Reply:
thanks